DataStudies/limitPlotScripts/set_limit.py

```diff
@@ -383,12 +443,12 @@
 # TPT.Draw()
 climits.RedrawAxis()
 
-CMS_lumi.extraText = 'Internal'
+CMS_lumi.extraText = '            Internal'
```

Comment:
I had to undo this, it's interesting that your plots look different than mine, maybe that I use an older version of ROOT, idk